### PR TITLE
Write tests for app, getEnvironmentVariable() and CommandFactory.loadCommands()

### PR DESCRIPTION
**Pull request by ajsaraujo**

### CODE REVIEW

File: test/factories/CommandFactoryTest.ts [hunk 1]
```diff
@@ -17,12 +19,53 @@ describe("CommandFactory", () => {
 		aliases = new MockCommandWithAlias().getAliases();
 
 		factory = new CommandFactory();
-		factory["commands"] = {};
-		factory["commands"][commandName] = () => new MockCommand();
-		factory["commands"][commandWithAliasesName] = () => new MockCommandWithAlias();
+		factory.commands = {};
+		factory.commands[commandName] = () => new MockCommand();
+		factory.commands[commandWithAliasesName] = () => new MockCommandWithAlias();
```
Comment (ajsaraujo) [inline test/factories/CommandFactoryTest.ts:25]: Oops. Only realized this now, I must have done it while using "Fix all auto-fixable problems". Let me know if you want me to undo it

Comment (LamboCreeper) [reply test/factories/CommandFactoryTest.ts:25]: Nah, `factory.commands` makes more sense then `factory["commands"]`.

**Review by LamboCreeper — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "The CodeSupport Discord bot",
   "main": "./build/app.js",
   "scripts": {
-    "start": "node -r dotenv/config ./build/app.js",
+    "start": "node -r dotenv/config ./build/index.js",
```
Comment (ajsaraujo) [inline package.json:7]: As I discussed with @Jason2605 in #discord-bot, `index.ts` was created to separate app definition and execution. If everything is in the same place, the app is going to be executed when a test imports it, which we don't want to happen.

File: test/app/appTest.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import { expect } from "chai";
```
Comment (LamboCreeper) [inline test/app/appTest.ts]: Why is this in a folder called `app`? That doesn't match `src/`'s structure.

**Review by LamboCreeper — APPROVED** [review]
Amazing. Thank you so much!

File: test/app/appTest.ts (hunk 3 above)
Comment (ajsaraujo) [reply test/app/appTest.ts]: When I put inside `test/`, the tests weren't ran. But now I realized that I had to tweak the test script, already fixed it. :+1:

**Review by LamboCreeper — APPROVED** [review]

**Pull request merged**